airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: e7778cfc-e97c-4458-9ecb-b4f2bba8946c
-  dockerImageTag: 2.1.5
```

Comment:
Don't think skipping a version causes any problems, but just for continuity's sake can we set the version to 2.1.6 here (as well as in pyproject and the changelog)? Otherwise all LGTM!

Reply:
I hear that, but all the tools individually were incrementing, so this was easier. Let's just roll forward.